go.mod

```diff
@@ -239,3 +239,5 @@ retract (
 	v0.53.2
 	v0.53.1 // Published accidentally with incorrect license in depdencies
 )
+
+replace github.com/anchore/packageurl-go => github.com/LaurentGoderre/packageurl-go v0.0.0-20231207160803-decab5854c64
```

Comment:
Only here for the tests to pass. Remove when https://github.com/anchore/packageurl-go/pull/16 lands

Reply:
I've removed the replace and used the raw value and left a comment. We try to keep as many replace directives out of the go.mod as possible.